plugins/woocommerce/changelog/53669-add-add-recurring-payments-icon

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: add
+
+Add "Recurring payments" icon next to payment providers
```

Comment:
```suggestion
Add a "Recurring payments" icon next to installed payment providers that support WooCommerce Susbscriptions.
```

Reply:
Newline character is missing at the end.

Reply:
It's added automatically, I've tried to remove it in the other pr - then CI removes it back 🤷